apps/backend/src/api/monitoring.py

```
@@ -0,0 +1,519 @@
+        # Remove None alerts
+        dashboard_data["alerts"] = [alert for alert in dashboard_data["alerts"] if alert is not None]
+        
+        return dashboard_data
+        
+    except Exception as e:
+        logger.error(f"Dashboard data collection failed: {e}", exc_info=True)
+        logger.debug(f"Full traceback: {traceback.format_exc()}")
+        raise HTTPException(
+            status_code=500,
+            detail="Dashboard data service temporarily unavailable. Please try again later."
+        ) from e
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)